Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM --platform=linux/amd64 crystallang/crystal:latest-alpine as builder
+
+RUN apk add --no-cache \
+  git \
+  make
+WORKDIR /root/src
+COPY . . 
+RUN make static_mt
```

Comment:
We could skip downloading packages by replacing make with the command `shards build --production --no-debug --release -Dpreview_mt --static` (not sure if make is already included in the crystal image, but since shards *should* be, git should exist too)